### ISSUE TITLE
Missing error massages

### ISSUE THREAD

**Issue by digitaldonkey** [danielnovy/dapp-poc-starter#1]
In some standard cases the App fails without providing the developer helpful information:

- If testrpc is not available there is no useful Error message. (Error is: OPTIONS http://localhost:8545/ net::ERR_CONNECTION_REFUSED)
- if Contract wasn't deployed there is no useful Error message. (Uncaught Error: new BigNumber() not a base 16 number)

**Comment by danielnovy** [comment]
Good point (sorry, just saw your msg now). I solved those problems in the project that used this boilerplate but forgot to update it. Will do it now. Tks!
